src/rai_extensions/rai_nomad/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>rai_nomad</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="kacper.dabrowski@robotec.ai">kdabrowski</maintainer>
+  <license>Apache-2.0</license>
+
```

Comment:
Please add the following dependencies:
- std_msgs
- geometry_msgs
- sensor_msgs